src/components/refresher/refresher.ts

```diff
@@ -202,6 +202,7 @@ export class Refresher {
 
   constructor(private _plt: Platform, @Host() private _content: Content, private _zone: NgZone, gestureCtrl: GestureController) {
     this._events = new UIEventManager(_plt);
+    _content._hasRefresher = true;
     _content.setElementClass('has-refresher', true);
```

Comment:
maybe this is a good oportunity to merge:
```
_content._hasRefresher = true;
_content.setElementClass('has-refresher', true);
```

to something like:
```
_content._setRefresher(this);
```

this way we reduce the coupling between both components a little bit, thoughts?